### PR TITLE
Add Stories support

### PR DESCRIPTION
**Pull request by dkstevekwak**
Stories checkbox
Screenshot 2024-03-12 at 1 24 05 PM
Stories video creation
Screenshot 2024-03-12 at 1 24 26 PM
Stories publishing
Screenshot 2024-03-12 at 1 24 55 PM
Stories published evidence
Screenshot 2024-03-12 at 1 25 25 PM
View stories
Screenshot 2024-03-12 at 1 26 04 PM
Reels publish still works as expected
Screenshot 2024-03-12 at 1 27 20 PM

### CODE REVIEW

**Comment by anujs3** [comment]
Do we plan on adding photo publishing support?

**Comment by dkstevekwak** [comment]
> Do we plan on adding photo publishing support?

Yes but in another PR in the future. This is in preparation for the video API summit which will mostly have the developers interested in the video use case

File: fb_reels_publishing_api_sample/index.js [hunk 1]
```diff
@@ -273,14 +279,26 @@ app.get("/listUploadedVideos", async function(req, res) {
                             if (value['updated_time']) {
                                 self[index]['updated_time'] = (new Date(value['updated_time'])).toLocaleString();
                             }
+                            if (isStories && value['creation_time']) {
+                                self[index]['creation_time'] = (new Date(value['creation_time'] * 1000)).toLocaleString();
+                            }
                         })
                         // Render the Upload page, but now send back the list of past reels of the selected page
```
Comment (kareem1234) [inline fb_reels_publishing_api_sample/index.js:286]: is there supposed to be a difference between these two branches under if (isStories) ?

Comment (dkstevekwak) [reply fb_reels_publishing_api_sample/index.js:286]: reels has a field called `updated_time` which is in milliseconds while stories has a field called `creation_time` which is in seconds. Thus you need to multiply it by 1000

**Review by kareem1234 — APPROVED** [review]

**Pull request merged**